src/main/java/com/bravo/user/dao/model/User.java

```diff
@@ -4,19 +4,11 @@
 import java.time.LocalDateTime;
 import java.util.List;
 import java.util.UUID;
-import javax.persistence.CascadeType;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.FetchType;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.OneToMany;
-import javax.persistence.OneToOne;
-import javax.persistence.Table;
+import javax.persistence.*;
+
 import lombok.Data;
 
 @Entity
-@Data
```

Comment:
Getters and Setters are provided by Lombok @DaTa annotation. Why did you remove it? https://projectlombok.org/features/Data